python/ray/actor.py

```diff
@@ -271,7 +272,21 @@ def _bind(
             options,
             other_args_to_resolve=other_args_to_resolve,
         )
-        return node
+
+        if node.num_returns > 1:
```

Comment:
```python3
            output_nodes: List[ClassMethodNode] = []
            for i in range(node.num_returns):
                output_node = ClassMethodNode(
                    f"return_idx_{i}",
                    (node, i),
                    dict(),
                    dict(),
                    {IS_CLASS_METHOD_OUTPUT_KEY: True},
                )
                output_nodes.append(output_node)
            return tuple(output_nodes)
```

can we just always make it return tuple and unify logics for num_returns > 1 and num_returns == 1?

Reply:
If `num_returns == 1`, returning a tuple requires the user to extract from this tuple. It would break the current usage, I suppose.

Reply:
For now, we return `ClassMethodNode` for `num_returns == 1` and `Tuple[ClassMethodNode, ...]` for `num_returns > 1`.